Header header
int32 command
int32 LOAD=1
int32 PLAY=2
int32 STOP=3
int32 PAUSE=4
---
Header header
bool success

